client/components/marketing-survey/cancel-purchase-form/step-components/educational-content-step.tsx

```diff
@@ -0,0 +1,333 @@
+							props.onDecline?.();
+						} }
+						isBusy={ isBusy }
+						disabled={ isBusy }
+					>
+						{ declineButtonText }
+					</Button>
+				</div>
+			</div>
+		</div>
+	);
+}
+
+type StepProps = {
+	type: UpsellType;
+	site: SiteDetails;
+	onDecline: () => void;
+	cancellationReason: string;
+};
+
+export default function EducationalCotnentStep( { type, site, ...props }: StepProps ) {
+	const translate = useTranslate();
+	const helpCenter = useDispatch( 'automattic/help-center' );
+
+	switch ( type ) {
+		case 'education:loading-time':
+			return (
+				<Content
+					image={ imgLoadingTime }
+					title={ translate( 'Loading times best practices' ) }
+					onDecline={ props.onDecline }
+				>
+					<p>
+						{ translate(
+							'A few common factors can slow down your site. You may want to try some of these fixes to improve your site’s loading time.'
+						) }
+					</p>
+					<ul className="cancel-purchase-form__edu-icon-list">
+						<li>
+							<div className="cancel-purchase-form__edu-icon">
+								<MaterialIcon icon="power" size={ 24 } />
+							</div>
+							<div>
+								{ translate(
+									'Uninstall any unnecessary plugins. Sites with too many third-party plugins installed can take longer to load.'
+								) }
+							</div>
+						</li>
+						<li>
+							<div className="cancel-purchase-form__edu-icon">
+								<MaterialIcon icon="palette" size={ 24 } />
+							</div>
+							<div>
+								{ translate(
+									'Make sure you’re using a fast theme. Some themes come packed with extra features that can slow down your site.'
+								) }
+							</div>
+						</li>
+						<li>
+							<div className="cancel-purchase-form__edu-icon">
+								<MaterialIcon icon="photo_library" size={ 24 } />
+							</div>
+							<div>
+								{ translate(
+									'Optimize your images before uploading them. Unnecessarily large image files take longer to load.'
+								) }
+							</div>
+						</li>
+						<li>
+							<div className="cancel-purchase-form__edu-icon">
+								<MaterialIcon icon="library_books" size={ 24 } />
+							</div>
+							<div>
+								{ translate(
+									'Read more about how to improve your site’s speed and performance {{link}}here{{/link}}.',
+									{
+										components: {
+											link: <Button href="https://wordpress.com/support/site-speed/" isLink />,
+										},
+									}
+								) }
+							</div>
+						</li>
+					</ul>
+				</Content>
+			);
+		case 'education:free-domain':
+			return (
+				<Content
+					image={ imgFreeDomain }
+					title={ translate( 'Step-by-step guide to get a free domain' ) }
+					onDecline={ props.onDecline }
+				>
+					<p>
+						{ translate(
+							'You can register a free domain name for one year with the purchase of any WordPress.com annual or two-year plan!'
+						) }
+					</p>
+					<ul>
+						<li>
+							{ translate(
+								'Go to Upgrades → Domains and click {{link}}Add a Domain{{/link}} to register your plan’s free domain',
```

Comment:
This might not quite apply if they're on a monthly plan and cancelling it but I think we can ship and iterate if this step turns out to be successful.

Reply:
cc @JanaMW27